src/registrar/templates/application_done.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block title %}Thank you for your domain request{% endblock %}
+
+{% block content %}
+<h1>Thank you!</h1>
+
+<p>
+Thank you for your domain request. We'll email a copy of your request to you,
+your authorizing official, and any contacts you added.</p>
+
+<h2>Next steps in this process</h2>
+
+<ul>
+
+    <li> We'll review your request. This could take up to two weeks. During
+        this review we'll verify that you're eligible for a .gov domain, that
+        your authorizing official approves your request, and that your domain
+        meets our naming requirements.
+    </li>
+
+    <li> You can <a href="/application/{{ application_id }}">check the
+            status</a> of your request at any time.
+    </li>
+
+    <li> We'll email you with any questions or when we complete our
+        review.</li>
+</ul>
+
+<h2>Option to enter domain name server information</h2>
+
+<p>Before your domain can be used we'll need information about your
+domain name servers. If you have this information you can enter it now.
+If you don't have it, that's okay. You can enter it later on the
+<a href="/domains">manage your domains page</a>.
+</p>
+{% endblock %}
```

Comment:
One small item left out is to a placeholder button "Enter DNS name servers" 
<img width="627" alt="image" src="https://user-images.githubusercontent.com/52677065/203163655-1a9f4050-1cd2-40ba-9203-a9f676b07efd.png">

Reply:
Added, thanks for catching that:
![Screen Shot 2022-11-22 at 11 28 17](https://user-images.githubusercontent.com/443389/203381568-1d36dbc0-ab54-42cc-97dc-319f0dfa247d.png)